guides/common/modules/ref_prerequisites-disaster-recovery-with-active-and-passive-project-server-and-backup-and-restore.adoc

```diff
@@ -0,0 +1,5 @@
+[id="prerequisites-disaster-recovery-with-active-and-passive-project-server-and-backup-and-restore"]
+= Prerequisites
+
+* Review xref:overview-of-recommended-disaster-recovery-plans[] to make sure that this disaster recovery plan works for you.
+* You have a {ProjectServer} installed.
```

Comment:
What does this mean? Do you mean that users need to have a secondary server installed?

I'd argue that almost every procdure requires a ProjectServer.

Reply:
This is meant to describe the initial situation we want the user to be in. In a way, "you need a server" also implies "you need a single server, but don't worry about your second server because you will create that later in the procedure" or "the server that you currently use is good enough for you to start this procedure".